### ISSUE TITLE
How can I do transfer learning based on your pre-trained DTI models

### ISSUE THREAD

**Issue by xuzhang5788** [kexinhuang12345/DeepPurpose#46]
Thank you so much for your great work. If I have a small amount of data (for example I only have 100 drug-protein pairs) for a specific problem, I can't train my data from scratch. So, I just want to know how I can use your pre-trained model for transfer learning. Many thanks

**Comment by kexinhuang12345** [comment]
Hi, thanks. You can simply load `model = DTI.model_pretrained(FILE_PATH)` and `model.train(train, val, test)`.

**Comment by xuzhang5788** [comment]
Hi again.
If run
model = DTI.model_pretrained(FILE_PATH) 
model.train(train, val, test)

I think that the model's config comes from the pre-trained model. However, for the transfer learning's purpose, we want to keep the pre-trained model's architecture and trained weights as initial weights for the new data, a new model can have different learning rates and epochs. So, before training, can I rewrite config? 

For transfer learning, usually do step by step, such as first unfreeze the last layer, train several epochs, then unfreeze more layers upper, train several epochs, finally, unfreeze the whole layers and train a couple of epochs to get the final results. I don't know how you performed this.

Thank a lot

**Comment by pykao** [comment]
Hi @kexinhuang12345 

I have the same question. If I would like to do the transfer learning. I am able to reload the pre-trained model use DTI.model_pretained(FILE_PATH) and then model.train(train.val, test). However, if I would like to change the learning weight or weight decay. What should I do?

**Comment by kexinhuang12345** [comment]
Hey @xuzhang5788 @pykao sorry for the late reply. I haven't used transfer learning before, but it should work by modifying the config file, such as `model.config['lr'] = 1e-3` and then run model.train(train, val, test). Since the train function utilizes the self.config variable
